admin_guide/assembly_replace-etcd-member.adoc

```diff
@@ -0,0 +1,28 @@
+[[assembly_replace-etcd-member]]
+= Replacing a failed etcd member
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:context: replace-etcd-member
+
+toc::[]
+
+
+In cases where etcd members have failed and you still have a quorum of etcd
+cluster members running, you can use the surviving members to
+add additional etcd members without downtime.
```

Comment:
This makes it sound like the surviving members are bringing them back to life. Would it be more accurate to say "You can use the surviving members as templates of your etcd configuration to add additional etcd members without downtime." or something? Am I misunderstanding how it works?

Reply:
I ... actually think both interpretations are valid. You have to run the commands from a surviving node and use files from it to make the new node. I take your point about node necromancy, though.

I'll use "If some etcd members fail, but you still have a quorum of etcd members, you can
use the remaining etcd members and the data that they contain to add more etcd
members without etcd or cluster downtime."